front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/Content/index.tsx

```diff
@@ -70,19 +70,18 @@ const CustomPagesEditContent = () => {
     //   titleMessageDescriptor: sectionToggleMessages.eventsList,
     //   tooltipMessageDescriptor: sectionToggleMessages.eventsListTooltip,
     // },
-    // next iteration!
-    // {
-    //   name: 'bottom_info_section_enabled',
-    //   titleMessageDescriptor: sectionToggleMessages.bottomInfoSection,
-    //   tooltipMessageDescriptor: sectionToggleMessages.bottomInfoSectionTooltip,
-    //   linkToPath: 'bottom-info-section',
-    // },
     {
       name: 'files_section_enabled',
       titleMessageDescriptor: sectionToggleMessages.attachmentsSection,
       tooltipMessageDescriptor: sectionToggleMessages.attachmentsSectionTooltip,
       linkToPath: 'attachments',
     },
```

Comment:
Moved it down because it's the last thing on the page, I think it makes sense for the toggles to match the order of the content